src/fastoad_cs25/models/loads/sizing_loads/gust.py

```diff
@@ -56,19 +57,18 @@ def setup(self):
         self.add_input("data:load_case:lc2:U_gust", val=np.nan, units="m/s")
         self.add_input("data:load_case:lc2:altitude", val=np.nan, units="ft")
         self.add_input("data:load_case:lc2:Vc_EAS", val=np.nan, units="m/s")
-        self.add_input("data:load_case:manoeuvre_load_factor", val=2.5)
         self.add_input("data:load_case:gust_intensity", val=1.0)
+        self.add_input("data:mission:sizing:cs25:safety_factor_1", val=1.5)
+        self.add_input("data:mission:sizing:cs25:safety_factor_2", val=1.5)
 
-        self.add_output("data:mission:sizing:cs25:load_factor_1")
-        self.add_output("data:mission:sizing:cs25:load_factor_2")
-        self.add_output("data:mission:sizing:cs25:sizing_load_1", units="kg")
-        self.add_output("data:mission:sizing:cs25:sizing_load_2", units="kg")
+        self.add_output("data:mission:sizing:cs25:gust:load_factor_1")
+        self.add_output("data:mission:sizing:cs25:gust:load_factor_2")
+        self.add_output("data:mission:sizing:cs25:gust:sizing_load_1", units="N")
+        self.add_output("data:mission:sizing:cs25:gust:sizing_load_2", units="N")
 
     def setup_partials(self):
         self.declare_partials("*", "*", method="fd")
```

Comment:
Two things here:
- I don't know which option is better but you can do that in the setup directly (which we do on FAST-OAD_CS23 and the hybrid electric version)
- Even if you don't actually compute them, you could defined which output depend on which inputs 😅

Reply:
You are totally right actually. OpenMDAO actually recommends not doing it inside the setup (https://openmdao.org/newdocs/versions/latest/advanced_user_guide/analytic_derivatives/partial_derivs_explicit.html)😅